### PR TITLE
Basic support for IPv4 static routes

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

File: sysrib/static.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		},
+	}
+}
+
+// monitorStaticRoutes starts a gothread to check for static route
+// configuration changes.
+// It returns an error if there is an error before monitoring can begin.
+func (s *Server) monitorStaticRoutes(yclient *ygnmi.Client) error {
+	b := &ocpath.Batch{}
+	staticroot := ocpath.Root().NetworkInstance(fakedevice.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "DEFAULT")
```
Comment (DanG100) [inline sysrib/static.go:63]: should this a const? configureable?

File: sysrib/static.go [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		context.Background(),
+		yclient,
+		b.Config(),
+		func(root *ygnmi.Value[*oc.Root]) error {
+			rootVal, ok := root.Val()
+			if !ok {
+				return ygnmi.Continue
+			}
+			staticp := rootVal.GetOrCreateNetworkInstance(fakedevice.DefaultNetworkInstance).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "DEFAULT")
+			if staticp == nil {
```
Comment (DanG100) [inline sysrib/static.go:84]: if you're calling GetOrCreate above when would this be nil?

File: sysrib/static.go (hunk 1 above)
Comment (wenovus) [reply sysrib/static.go]: Done.

File: sysrib/static.go (hunk 2 above)
Comment (wenovus) [reply sysrib/static.go]: Removed.

**Review by DanG100 — APPROVED** [review]

**Pull request merged**